config/routes.rb

```diff
@@ -29,14 +29,13 @@
     end
   end
 
-  namespace :api do
+  namespace :api, defaults: { format: "json" } do
     resources :school_search, only: %i[index]
     resource :notify_callback, only: :create, path: "notify-callback"
 
     namespace :v1 do
-      unless Rails.env.staging? || Rails.env.production?
-        resources :users, only: :index
-      end
+      resources :early_career_teacher_participants, only: %i[create], path: "early-career-teacher-participants"
```

Comment:
Not convinced by the naming here. Maybe `participants/early-career-teacher`?

Reply:
Renamed to ```participants/induction-progress``` in 5416057
https://ecf-review-pr-333.london.cloudapps.digital/api-docs/index.html?urls.primaryName=API%20V1%20Docs